packages/core/bootstrap/src/lib/external-adapter/validator.ts

```diff
@@ -118,6 +169,29 @@ export class Validator {
     return multiple
   }
 
+  overrideToken = (symbol: string, network = 'ethereum'): string | undefined => {
+    if (!this.validated.tokenOverrides) return undefined
+    return this.validated.tokenOverrides.get(network.toLowerCase())?.get(symbol.toLowerCase())
+  }
+
+  overrideIncludes = (adapter: string, from: string, to: string, includes: Includes[]): Includes | undefined => {
+    includes.filter(include =>
+      (include.from.toLowerCase() === from.toLowerCase() && include.to.toLowerCase() === to.toLowerCase()) &&
+      (!include.adapters || include.adapters.map(adapter => adapter.toLowerCase()).includes(adapter.toLowerCase()))
+    )[0]
+    // Search through `presetIncludes` to find matching override for adapter and to/from pairing.
+    const pairs = presetIncludes.filter(pair => pair.from === from && pair.to === to)
```

Comment:
Gotta normalize these

```suggestion
    const pairs = presetIncludes.filter(pair => pair.from.toLowerCase() === from.toLowerCase() && pair.to.toLowerCase() === to.toLowerCase())
```